dotCMS/src/main/java/com/dotmarketing/startup/runalways/Task00001LoadSchema.java

```diff
@@ -111,9 +120,12 @@ public boolean forceRun() {
 			conn =DbConnectionFactory.getConnection();
 		}
 		catch(DotRuntimeException dre){
-			Logger.fatal(this.getClass(),"Unable to get dotCMS database connection.  Please change your connection properties and restart");
-			Logger.fatal(this.getClass(),"Unable to get dotCMS database connection.  Please change your connection properties and restart");
-			Logger.fatal(this.getClass(),"Unable to get dotCMS database connection.  Please change your connection properties and restart");
+			Logger.fatal(this.getClass(),"Unable to get the dotCMS database connection. Please " +
+					"change your connection properties and restart");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "change your connection properties and restart" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)